test-unit.gemspec

```diff
@@ -31,7 +31,9 @@ Gem::Specification.new do |spec|
   spec.test_files += Dir.glob("test/**/*")
 
   spec.metadata = {
-    "source_code_uri" => "https://github.com/test-unit/test-unit"
+    "source_code_uri" => "https://github.com/test-unit/test-unit",
+    "documentation_uri" => "https://rubydoc.info/github/test-unit/test-unit",
```

Comment:
Is https://test-unit.github.io/test-unit/en/ the best one? 😮 

<img width="987" alt="screen_shot 2021-06-06 22 05 07" src="https://user-images.githubusercontent.com/1180335/120925366-48865000-c713-11eb-9d0e-df6cba1ed46d.png">


```suggestion
    "documentation_uri" => "https://test-unit.github.io/test-unit/en/",
```